libsolidity/ast/AST.h

```diff
@@ -508,7 +508,7 @@ class ContractDefinition: public Declaration, public StructurallyDocumented, pub
 	std::vector<ModifierDefinition const*> functionModifiers() const { return filteredNodes<ModifierDefinition>(m_subNodes); }
 	std::vector<FunctionDefinition const*> definedFunctions() const { return filteredNodes<FunctionDefinition>(m_subNodes); }
 	std::vector<EventDefinition const*> events() const { return filteredNodes<EventDefinition>(m_subNodes); }
-	std::vector<EventDefinition const*> const& interfaceEvents() const;
+	std::vector<EventDefinition const*> const& interfaceEvents(bool _ignoreErrors = false) const;
```

Comment:
Can you explain what ignoreerrors mean here?

Reply:
Done.